src/modelfree/training/victim_envs.py

```diff
@@ -1,12 +1,27 @@
+import numpy as np
+
 from aprl.envs.multi_agent import VecMultiWrapper, _tuple_pop, _tuple_space_filter
 
 
 class EmbedVictimWrapper(VecMultiWrapper):
     """Embeds victim in a (Transparent)CurryVecEnv. Also takes care of closing victim's session"""
-    def __init__(self, multi_env, victim, victim_index, transparent, deterministic):
-        self.victim = victim
-        cls = TransparentCurryVecEnv if transparent else CurryVecEnv
-        curried_env = cls(multi_env, self.victim, agent_idx=victim_index,
+    def __init__(self, multi_env, victims, victim_index, transparent, deterministic):
+        multi_victim = len(victims) > 0
```

Comment:
This should presumably be `len(victims) > 1`, but moot point if we're merging them.